app/views/comments/_edited_notice.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if decorated_comment.display_edited? %>
+  <div class="comment-edited-notice comment-date">
+    Edited
+    <time datetime="<%= decorated_comment.edited_timestamp %>">
+      <%= decorated_comment.edited_at %>
```

Comment:
I still think we should apply this change as well https://github.com/thepracticaldev/dev.to/pull/4701#discussion_r342460216 - we tend to the date in that format around the website, we might as well be consistent